ui/src/router/details/logs/RouterLogsView.js

```diff
@@ -55,24 +67,73 @@ export const RouterLogsView = ({ router }) => {
     (entries) =>
       !!entries.length ? entries[entries.length - 1].timestamp : undefined,
     (entries) => entries.map((entry) => LogEntry.fromJson(entry).toString()),
-    configOptions
+      appConfig.podLogs,
   );
 
   const availableComponents = components.filter(
     (c) => c.value === "router" || !!get(router, `config.${c.value}`)
   );
 
+  useEffect(
+    () => {
+      let urls = {}
+      if (appConfig.imagebuilder.cluster && appConfig.imagebuilder.gcp_project && appConfig.imagebuilder.namespace &&
+        currentProject) {
+        // set router url
+        urls["router"] = createStackdriverUrl({
+          gcp_project: environment.gcp_project,
+          cluster: environment.cluster,
+          namespace: currentProject.name,
+          pod_name: router.name + "-turing-router-"  + router.config.version,
```

Comment:
This naming convention deliberately ignores the knative revision number (`0` by default if deployed by the API server, anything else if the Knative spec is modified and saved manually):

Example of a pod name:
```test-router-1-turing-router-9-0-deployment-6f456745f7-xdr8x````